orm-poc/src/main/java/com/orm/poc/mapper/EntityMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.orm.poc.mapper;
+
+import com.orm.poc.utils.ReflectionUtils;
+
+import java.lang.reflect.Field;
+import java.sql.ResultSet;
+
+public class EntityMapper<T> {
```

Comment:
@mvorotniak there is no need to declare type parameter as the class level and then create separate instances for each entity. Just make `mapToEntity` generic and use the same method for all entities.

Reply:
Thank you! Will fix